services/powershellgallery/powershellgallery.tester.js

```diff
@@ -36,170 +27,28 @@ t.create('total downloads (not found)')
   .get('/dt/not-a-real-package.json')
   .expectJSON({ name: 'downloads', value: 'not found' })
 
-t.create('total downloads (connection error)')
-  .get('/dt/ACMESharp.json')
-  .networkOff()
-  .expectJSON({ name: 'downloads', value: 'inaccessible' })
-
-t.create('total downloads (unexpected response)')
-  .get('/dt/ACMESharp.json')
-  .intercept(nock =>
-    nock('https://msconfiggallery.cloudapp.net')
-      .get(
-        '/api/v2/Packages()?$filter=Id%20eq%20%27ACMESharp%27%20and%20IsLatestVersion%20eq%20true'
-      )
-      .reply(invalidJSON)
-  )
-  .expectJSON({ name: 'downloads', value: 'invalid' })
-
-// version
-
 t.create('version (valid)')
   .get('/v/ACMESharp.json')
   .expectJSONTypes(
     Joi.object().keys({
-      name: 'powershellgallery',
+      name: 'powershell gallery',
       value: isVPlusDottedVersionNClauses,
     })
   )
 
-t.create('version (mocked, yellow badge)')
-  .get('/v/ACMESharp.json?style=_shields_test')
-  .intercept(nock =>
-    nock('https://msconfiggallery.cloudapp.net')
-      .get(
-        '/api/v2/Packages()?$filter=Id%20eq%20%27ACMESharp%27%20and%20IsLatestVersion%20eq%20true'
-      )
-      .reply(200, nuGetV2VersionJsonWithDash)
-  )
-  .expectJSON({
-    name: 'powershellgallery',
-    value: 'v1.2-beta',
-    colorB: colorscheme.yellow.colorB,
-  })
-
-t.create('version (mocked, orange badge)')
-  .get('/v/ACMESharp.json?style=_shields_test')
-  .intercept(nock =>
-    nock('https://msconfiggallery.cloudapp.net')
-      .get(
-        '/api/v2/Packages()?$filter=Id%20eq%20%27ACMESharp%27%20and%20IsLatestVersion%20eq%20true'
-      )
-      .reply(200, nuGetV2VersionJsonFirstCharZero)
-  )
-  .expectJSON({
-    name: 'powershellgallery',
-    value: 'v0.35',
-    colorB: colorscheme.orange.colorB,
-  })
-
-t.create('version (mocked, blue badge)')
-  .get('/v/ACMESharp.json?style=_shields_test')
-  .intercept(nock =>
-    nock('https://msconfiggallery.cloudapp.net')
-      .get(
-        '/api/v2/Packages()?$filter=Id%20eq%20%27ACMESharp%27%20and%20IsLatestVersion%20eq%20true'
-      )
-      .reply(200, nuGetV2VersionJsonFirstCharNotZero)
-  )
-  .expectJSON({
-    name: 'powershellgallery',
-    value: 'v1.2.7',
-    colorB: colorscheme.blue.colorB,
```

Comment:
We should probably replace all these deleted tests with versions that use XML fixtures.

Reply:
I’m not sure they’re needed. It seems that the only difference between these and the main tests is the coverage of the color logic. All the plumbing of those values is the same.

A much simpler way to test the color logic would be a unit test on the render function, and meanwhile it’s covered by tests on other nuget services.

Reply:
Fair point - previously it was all such a big ball o' mud that we couldn't test that logic independently